tests/bbs_plus.rs

```diff
@@ -48,7 +61,7 @@ pub fn bbs_sign_tests() {
         messages,
     };
     let js_value = serde_wasm_bindgen::to_value(&request).unwrap();
-    let s_res = bbs_sign(js_value);
+    let s_res = bbs_sign(js_value).await;
     assert!(s_res.is_ok());
```

Comment:
These checks could be dropped probably. The unwrap will panic if the assert isn't correct. The only advantage I can think up why this might be better is for formatting of tests, but I haven't ran them to check what the difference would be.

Reply:
Yeah the mike added the explicit assert statements I'll ask him about test structure later as a matter of proper styling